README.md

```diff
@@ -188,6 +194,7 @@ Please read the [contribution guidelines](CONTRIBUTING.md) if you want to contri
 - [WebAssembly overview (2016)](https://www.slideshare.net/ValeriiaMaliarenko/web-assembly-overview-by-mikhail-sorokovsky)
 - [Emscripten and WebAssembly (2015)](https://kripken.github.io/talks/wasm.html)
 - [An intro to WebAssembly (2015)](https://www.slideshare.net/danlbudden/an-introduction-to-webassembly)
+- [Web assembly overview by Mikhail Sorokovsky](https://www.slideshare.net/ValeriiaMaliarenko/web-assembly-overview-by-mikhail-sorokovsky)
```

Comment:
Unfortunately, this resource has already been defined some lines above (WebAssembly overview (2016)), I think that we can remove this line